packages/pds/src/services/index.ts

```diff
@@ -15,14 +15,27 @@ export function createServices(resources: {
   imgUriBuilder: ImageUriBuilder
 }): Services {
   const { messageQueue, blobstore, imgUriBuilder } = resources
-  return {
-    actor: ActorService.creator(),
-    auth: AuthService.creator(),
-    feed: FeedService.creator(imgUriBuilder),
-    record: RecordService.creator(messageQueue),
-    repo: RepoService.creator(messageQueue, blobstore),
-    admin: AdminService.creator(),
+  const services = {
+    get actor() {
+      return ActorService.creator(services)
```

Comment:
whoa actually didn't know that this scoping worked lol

Reply:
Haha yeah!  The getter makes access to `services` lazy, so it's being referenced only when someone indexes into `services`.  It's a pretty useful tool for creating app contexts!